modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/HeaderInformation.java

```diff
@@ -0,0 +1,822 @@
+	}
+
+	/**
+	 * Appends multiple {@link Integer} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Integer} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Integer... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Long} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Long} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Long... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Short} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Short} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Short... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Byte} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Byte} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Byte... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Double} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Double} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Double... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Appends multiple {@link Boolean} values assigned to the key.
+	 *
+	 * @param key    the key to which the values are assigned
+	 * @param values the new {@link Boolean} values that you want to append to the key
+	 * @return a reference to {@code this}, so the API can be used fluently
+	 * @see MultiMap#add(String, Iterable)
+	 */
+	public HeaderInformation add(String key, Boolean... values) {
+		return add(key, Arrays.stream(values).map(String::valueOf));
+	}
+
+	/**
+	 * Intermediate step to append a stream of strings to the wrapped {@link MultiMap Vert.x headers}.
```

Comment:
```suggestion
	 * Intermediate step to append a stream of strings to the wrapped {@link MultiMap Vert.x headers}.
	 * Returns a reference to {@code this} for fluent design.
	 *
	 * @param key      the key to which the values should be assigned
	 * @param stream   stream of values to assign to the given key
	 * @return a reference to {@code this}, so the API can be used fluently
```

Reply:
And maybe consider putting it to the end of the file. In Java you can put private methods to the eof and they usually aren't interesting for users of the API. This is why it is useful to put them to the very bottom of a class.